backend/controllers/orgUsersController.js

```diff
@@ -43,7 +43,7 @@ const createOrgUser = async (req, res) => {
 };
 
 const findOrgUserById = async (req, res) => {
-  const orgUserId = req.params.org_user_id;
+  const orgUserId = req.params;
```

Comment:
ooo we would still need `req.params.org_user_id` to find the id please

Reply:
we absolutely do thanks thuy